storage/inchi/src/test/java/org/openscience/cdk/inchi/AuxInfoToStructureTest.java

```diff
@@ -0,0 +1,109 @@
+package org.openscience.cdk.inchi;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openscience.cdk.exception.CDKException;
+import org.openscience.cdk.interfaces.IAtom;
+import org.openscience.cdk.interfaces.IAtomContainer;
+import org.openscience.cdk.interfaces.IStereoElement;
+import org.openscience.cdk.silent.SilentChemObjectBuilder;
+import org.openscience.cdk.stereo.TetrahedralChirality;
+import org.openscience.cdk.test.CDKTestCase;
+
+import java.util.Collection;
+
+/**
+ * Test case for {@link AuxInfoToStructure} class.
+ */
+public class AuxInfoToStructureTest extends CDKTestCase {
```

Comment:
The `extends CDKTestCase` is historic. If you do not use any of the functionality of the class, you can leave this out.